pkg/README.md

```diff
@@ -1,46 +1,82 @@
-# Grafana backend codebase
+# Backend
 
-The code should follow the [Backend style guide](/contribute/style-guides/backend.md).
+This directory contains the code for the Grafana backend. This document gives an overview of the directory structure, and ongoing refactorings.
 
-Refer to [architecture](ARCHITECTURE.md) for a description of the backend codebase architecture.
+For more information on developing for the backend:
+
+- [Backend style guide](/contribute/style-guides/backend.md)
+- [Architecture](/contribute/architecture)
+
+## Central folders of Grafanas backend
+
+| folder | description |
+| ------- | ----------- |
+| /pkg/api | Http Handlers and routing. Almost all handler funcs are global which is something we would like to improve in the future. Handlers should be assosicated with a struct that refers to all depedencies. Ex bus. |
```

Comment:
Spelling: 
- associated
- dependencies

Not sure I understand the last two words: `Ex bus`.

Reply:
```suggestion
| /pkg/api | Http Handlers and routing. Almost all handler funcs are global which is something we would like to improve in the future. Handlers should be associated with a struct that refers to all dependencies, for example, bus. |
```